garage.com/src/test/resources/CssSelectorsProject.rtf

```diff
@@ -0,0 +1,37 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2580
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\froman\fcharset0 Times-Roman;}
+{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red11\green76\blue180;}
+{\*\expandedcolortbl;;\cssrgb\c0\c0\c0;\cssrgb\c1961\c38824\c75686;}
+\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\deftab720
+\pard\pardeftab720\sa240\partightenfactor0
+
+\f0\fs29\fsmilli14667 \cf2 \expnd0\expndtw0\kerning0
+All problems are located on the site \cf3 http://www.demoqa.com/\cf2 . Use CSS Selectors. Use the shortest form possible. Avoid locating by index or spatial position unless otherwise noted. \
+1. Page: /text-box. Write a selector that will return the \'91Full Name\'92 textbox using the ID attribute. Hint: This element has the \'91input\'92 tag. \
+  R. $("input#userName")\
+2. Page: /buttons. Write a selector that will return all 3 buttons shown below. 
+\fs24 \
+  R.$("button.btn.btn-primary")\
+
+\fs29\fsmilli14667 3. Page: /buttons. Write a selector that will return the span element in the footer. 
+\fs24 \
+     R. $("footer > span")\
+
+\fs29\fsmilli14667 4. Page: /menu. Write a selector that will return the 3 list item elements in the unordered list. 
+\fs24 \
+R.$("ul#nav >li")\
+5.
+\fs29\fsmilli14667 Page: /select-menu. Write a selector that will get the option element where value = 4. 
+\fs24 \
+     R. $(\'93option[value = '4']")\
```

Comment:
Ensure that you narrow the scope to the menu element
div#selectMenuContainer option[value="4"]